src/store/mutations.ts

```diff
@@ -1,7 +1,50 @@
 import { RootState } from './RootState';
+import QueryBuilderSparqlGenerator from '@/sparql/QueryBuilderSparqlGenerator';
+import SparqlGenerator from '@/sparql/SparqlGenerator';
 
 export default {
 	setValue( state: RootState, value: string ): void {
 		state.value = value;
+	},
+	runQuery( state: RootState ): void {
+		const prefixes = {
+			wdt: 'http://www.wikidata.org/prop/direct/'
+		};
+		const queryObject = {
```

Comment:
This definitely needs a builder a dedicated types but that's for later, I made https://phabricator.wikimedia.org/T264898 and https://phabricator.wikimedia.org/T264897 for it.